kalite/distributed/static/js/distributed/exercises/perseus-helpers.js

```diff
@@ -1,24 +1,38 @@
 // $(Exercises).trigger("clearExistingProblem");
 
-var KhanUtil = window.KhanUtil || {
-    debugLog: function() {}
+var icu = {
```

Comment:
Variable now needed for some internationalisation functions. This needs to be modified for i18n compatibility.